src/app/components/timeline/CardDialog.js

```diff
@@ -57,9 +70,62 @@ class CardDialog extends Component {
     this.props.closeDialog()
   }
 
+  addInputRef = (ref) => {
+    // Set state needs to be queued so that React does not overwrite
+    // the attribute in a merge
+    setTimeout(() => {
+      this.setState({
+        inputRefs: [...this.state.inputRefs, ref],
+      })
+    }, 0)
+  }
```

Comment:
This adds a ref to the state of the parent component.  The ref points at the input which edits the custom attribute.